mucommander-core/src/main/java/com/mucommander/ui/action/MuAction.java

```diff
@@ -17,16 +17,22 @@
 
 package com.mucommander.ui.action;
```

Comment:
whether is a good place for that? I like that it is a single point, but on the other hand MuAction starts to get heavy . Move the new functionality to other class and call it from MuAction?

Reply:
the entire file has changed so it's hard to say.. what changed, just `displayKeyboardNotification` and `actionPerformed`?

Reply:
yes, however, if you select with shift all commits except the first one (MuAction refactor) they will see the exact changes.